src/libtrx/game/game_flow/common.c

```diff
@@ -231,6 +231,11 @@ const GF_LEVEL *GF_GetLevelBefore(const GF_LEVEL *const level)
     return nullptr;
 }
 
+bool GF_IsGymAvailable(void)
+{
+    return GF_GetGymLevel() != nullptr;
+}
```

Comment:
IMO it's a bit superfluous. I think not long ago we have removed this getter in favor of direct comparisons.

Reply:
Thanks, simplified.